package.json

```diff
@@ -18,7 +18,7 @@
     "@mate-academy/cypress-tools": "^1.0.4",
     "@mate-academy/eslint-config-react": "0.0.7",
     "@mate-academy/eslint-config-react-typescript": "^1.0.1",
-    "@mate-academy/scripts": "^1.2.1",
+    "@mate-academy/scripts": "^1.2.3",
     "@mate-academy/students-ts-config": "*",
     "@mate-academy/stylelint-config": "*",
     "@types/node": "^17.0.23",
```

Comment:
***Optional**
You could use the single adding file to the stage to avoid adding unnecessary files to the Git repo.
git add “path to file” instead of "git add ."
Or just add files directly
![image](https://user-images.githubusercontent.com/81115131/184652955-1e769922-ef07-4ed6-8027-69a738cb2e7c.png)